clients/fides-js/src/lib/initialize.ts

```diff
@@ -430,6 +433,26 @@ export const initialize = async ({
           overrides?.experienceTranslationOverrides
         );
 
+        // Provide the modal link label function to the client based on the current locale unless specified via props.
+        getModalLinkLabel = (props) => {
+          let modalLinkText = DEFAULT_MODAL_LINK_LABEL;
+          if (!props?.disableLocalization) {
+            if (i18n.t("exp.modal_link_label") !== "exp.modal_link_label") {
+              modalLinkText = i18n.t("exp.modal_link_label");
+            }
+          } else {
+            const defaultLocale = i18n.getDefaultLocale();
+            const defaultLocaleLabel =
+              window.Fides.experience?.experience_config?.translations.find(
```

Comment:
let's use `effectiveExperience` instead of the `window` obj as we can't guarantee that this is the most up-to-date.